src/main/java/net/mcreator/ui/help/HelpContextWithEntry.java

```diff
@@ -22,21 +22,8 @@
 import java.net.URI;
 import java.util.function.Supplier;
 
-public class HelpContextWithEntry implements IHelpContext {
-
-	@Nullable private final String name;
-	@Nullable private final URI contextURL;
-
-	@Nullable private final String entry;
-	@Nullable private final Supplier<?>[] arguments;
-
-	public HelpContextWithEntry(@Nullable String name, @Nullable URI contextURL, @Nullable String entry,
-			@Nullable Supplier<?>... arguments) {
-		this.name = name;
-		this.contextURL = contextURL;
-		this.entry = entry;
-		this.arguments = arguments;
-	}
+public record HelpContextWithEntry(@Nullable String name, @Nullable URI contextURL, @Nullable String entry,
+								   @Nullable Supplier<?>[] arguments) implements IHelpContext {

```

Comment:
Getters in this class could be replaced with record accessors too. Just change name to contextName in constructor and you are set

Reply:
Nevermind, it implements interface that defines these, it is ok.